.github/workflows/test_build.yml

```diff
@@ -85,7 +85,7 @@ jobs:
           sudo apt-get install -y -q --no-install-recommends libboost-stacktrace-dev \
                   libncurses-dev libncurses5-dev libreadline-dev libeigen3-dev liblapack-dev libxml2-dev \
                   libgc-dev libgdbm-dev libglpk-dev libgmp3-dev libgtest-dev libmpfr-dev libmpfi-dev libntl-dev gfan \
-                  libgivaro-dev libboost-regex-dev fflas-ffpack libflint-dev libmps-dev libfrobby-dev \
+                  libgivaro-dev libboost-regex-dev fflas-ffpack libflint-dev libflint-arb-dev libmps-dev libfrobby-dev \
```

Comment:
Why not just resign to using the ppa package for flint and factory?

Reply:
Mostly laziness.  :)

I deleted the FLINT package from the PPA because of the `BeginningMacaulay2` example bug.  It would likely be fixed by packaging factory for the PPA, but I haven't done that.  I figure that we'll likely be switching to Ubuntu 24.04  soon enough that I'm not sure if it's worth the effort.